recipe-server/client/control/components/extensions/CreateExtensionPage.js

```diff
@@ -19,7 +20,7 @@ import {
   },
 )
 @autobind
-export default class CreateExtensionPage extends React.PureComponent {
```

Comment:
The issue between `Pure/Components` is with regard to Ant's form/error handling. Ant uses plain JS objects for the `form` and `formErrors` props, which doesn't play well with `PureComponent`.

This page, however, shouldn't have been switched back to `Component`. I've gone through and corrected this. As for documenting when to use `Pure` vs regular components, I'd prefer to place that in some sort of centralized location, versus including the comment in each of the ~5 files. Any suggestions on that?

Reply:
It would be good to have guidelines in the docs like "use pure components where possible" and "watch out for these known issues". That provides a good source of truth and a better onboarding experience.

However, I think also putting comments next to the components reminding us of the hard to see problems of pure components in some areas would also be good. This would prevent people from changing them in the future during refactoring. It would also ease code review, so I stop asking dumb questions like this. 😄